lib/network/options.js

```diff
@@ -670,6 +675,10 @@ let configureOptions = {
     minVelocity: [0.1, 0.01, 0.5, 0.01],
     solver: ['barnesHut', 'forceAtlas2Based', 'repulsion', 'hierarchicalRepulsion'],
     timestep: [0.5, 0.01, 1, 0.01],
+    wind: {
+      x: [0, -10, 10, 0.1],
+      y: [0, -10, 10, 0.1]
+    },
```

Comment:
Playing with this it seems that 2 is already a really strong wind, 10 is insane (the nodes just fly off the screen immediately and the edges get really stretched out). I think this would be much better with smaller range and finer steps. I think most people would use wind +-0.x anyway.

PS: As always I may be wrong. If these strong winds have valid use cases feel free to correct me.
PPS: It would be nice to have an example with some fixed nodes and wind sliders to play with. In the existing physics example it just drifts off the screen.

Reply:
The size of the "wind" force required to move nodes is related to countering the forces applied by the "spring" edges pulling from fixed nodes.
In a network with three nodes, a 0.1 wind will do.
When there are more fixed nodes, a stronger force is needed.

Added a "wind.html" under examples/physics